### DISCUSSION TITLE
How do I enable to the component model feature?

### DISCUSSION THREAD

**Discussion by ctaggart** [wasmerio/wasmer#5276] [General]
The error says to enable the compont model feature. How do I enable to the component model feature? I did not see an option listed from `wasmer run --help`.
 
```
~/ms/StarlingMonkey15/test-rs> wasmer run ./target/wasm32-wasip1/debug/test-rs-composed.wasm
error: Unable to compile "./target/wasm32-wasip1/debug/test-rs-composed.wasm"
╰─▶ 1: compile error: Validate("unknown binary version and encoding combination: 0xd and 0x1, note: encoded as a component but the WebAssembly component model feature is not enabled - enable the feature to allow component validation (at offset 0x0)")
```

**Comment by syrusakbary** [comment]
Hey @ctaggart , Wasmer currently doesn't support the component model.

I see that you are using StarlinkMonkey. If you want to use something similar that already works in Wasmer, I recommend taking a look on [WinterJS](https://winterjs.org). Hope this helps!
